### ISSUE TITLE
2024-12-06, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#12]
# Today's Arxiv Papers

(1) [p-MoD: Building Mixture-of-Depths MLLMs via Progressive Ratio Decay](http://arxiv.org/abs/2412.04449v1), 2024-12-05

(2) [EgoPlan-Bench2: A Benchmark for Multimodal Large Language Model Planning in Real-World Scenarios](http://arxiv.org/abs/2412.04447v1), 2024-12-05

(3) [Moto: Latent Motion Token as the Bridging Language for Robot Manipulation](http://arxiv.org/abs/2412.04445v1), 2024-12-05

(4) [Divot: Diffusion Powers Video Tokenizer for Comprehension and Generation](http://arxiv.org/abs/2412.04432v1), 2024-12-05

(5) [Grounding Descriptions in Images informs Zero-Shot Visual Recognition](http://arxiv.org/abs/2412.04429v1), 2024-12-05

(6) [Florence-VL: Enhancing Vision-Language Models with Generative Vision Encoder and Depth-Breadth Fusion](http://arxiv.org/abs/2412.04424v1), 2024-12-05

(7) [Targeting the Core: A Simple and Effective Method to Attack RAG-based Agents via Direct LLM Manipulation](http://arxiv.org/abs/2412.04415v1), 2024-12-05

(8) [Discriminative Fine-tuning of LVLMs](http://arxiv.org/abs/2412.04378v1), 2024-12-05

(9) [Challenges in Trustworthy Human Evaluation of Chatbots](http://arxiv.org/abs/2412.04363v1), 2024-12-05

(10) [RMD: A Simple Baseline for More General Human Motion Generation via Training-free Retrieval-Augmented Motion Diffuse](http://arxiv.org/abs/2412.04343v1), 2024-12-05

(11) [Retrieval-Augmented Machine Translation with Unstructured Knowledge](http://arxiv.org/abs/2412.04342v1), 2024-12-05

(12) [Liquid: Language Models are Scalable Multi-modal Generators](http://arxiv.org/abs/2412.04332v1), 2024-12-05

(13) [The Hyperfitting Phenomenon: Sharpening and Stabilizing LLMs for Open-Ended Text Generation](http://arxiv.org/abs/2412.04318v1), 2024-12-05

(14) [FlashSloth: Lightning Multimodal Large Language Models via Embedded Visual Compression](http://arxiv.org/abs/2412.04317v1), 2024-12-05

(15) [Densing Law of LLMs](http://arxiv.org/abs/2412.04315v1), 2024-12-05

(16) [ALMA: Alignment with Minimal Annotation](http://arxiv.org/abs/2412.04305v1), 2024-12-05

(17) [Evolutionary Pre-Prompt Optimization for Mathematical Reasoning](http://arxiv.org/abs/2412.04291v1), 2024-12-05

(18) [Arabic Stable LM: Adapting Stable LM 2 1.6B to Arabic](http://arxiv.org/abs/2412.04277v1), 2024-12-05

(19) [PoTable: Programming Standardly on Table-based Reasoning Like a Human Analyst](http://arxiv.org/abs/2412.04272v1), 2024-12-05

(20) [SynFinTabs: A Dataset of Synthetic Financial Tables for Information and Table Extraction](http://arxiv.org/abs/2412.04262v1), 2024-12-05

(21) [VASCAR: Content-Aware Layout Generation via Visual-Aware Self-Correction](http://arxiv.org/abs/2412.04237v1), 2024-12-05

(22) [Addressing Hallucinations with RAG and NMISS in Italian Healthcare LLM Chatbots](http://arxiv.org/abs/2412.04235v1), 2024-12-05

(23) [A Context-aware Framework for Translation-mediated Conversations](http://arxiv.org/abs/2412.04205v1), 2024-12-05

(24) [AL-QASIDA: Analyzing LLM Quality and Accuracy Systematically in Dialectal Arabic](http://arxiv.org/abs/2412.04193v1), 2024-12-05

(25) [Leveraging Large Language Models to Generate Course-specific Semantically Annotated Learning Objects](http://arxiv.org/abs/2412.04185v1), 2024-12-05

(26) [SKIM: Any-bit Quantization Pushing The Limits of Post-Training Quantization](http://arxiv.org/abs/2412.04180v1), 2024-12-05

(27) [Bench-CoE: a Framework for Collaboration of Experts from Benchmark](http://arxiv.org/abs/2412.04167v1), 2024-12-05

(28) [Reducing Tool Hallucination via Reliability Alignment](http://arxiv.org/abs/2412.04141v1), 2024-12-05

(29) [Monet: Mixture of Monosemantic Experts for Transformers](http://arxiv.org/abs/2412.04139v1), 2024-12-05

(30) [Enhancing Mathematical Reasoning in LLMs with Background Operators](http://arxiv.org/abs/2412.04110v1), 2024-12-05

(31) [Pre-train, Align, and Disentangle: Empowering Sequential Recommendation with Large Language Models](http://arxiv.org/abs/2412.04107v1), 2024-12-05

(32) [Practical Considerations for Agentic LLM Systems](http://arxiv.org/abs/2412.04093v1), 2024-12-05

(33) [LossAgent: Towards Any Optimization Objectives for Image Processing with LLM Agents](http://arxiv.org/abs/2412.04090v1), 2024-12-05

(34) [Towards Generalizable Autonomous Penetration Testing via Domain Randomization and Meta-Reinforcement Learning](http://arxiv.org/abs/2412.04078v1), 2024-12-05

(35) [Does your model understand genes? A benchmark of gene properties for biological and text models](http://arxiv.org/abs/2412.04075v1), 2024-12-05

(36) [ProtDAT: A Unified Framework for Protein Sequence Design from Any Protein Text Description](http://arxiv.org/abs/2412.04069v1), 2024-12-05

(37) [From Code to Play: Benchmarking Program Search for Games Using Large Language Models](http://arxiv.org/abs/2412.04057v1), 2024-12-05

(38) [Prompt Engineering Guidance for Conceptual Agent-based Model Extraction using Large Language Models](http://arxiv.org/abs/2412.04056v1), 2024-12-05

(39) [Hostility Detection in UK Politics: A Dataset on Online Abuse Targeting MPs](http://arxiv.org/abs/2412.04046v1), 2024-12-05

(40) [GenChaR: A Dataset for Stock Chart Captioning](http://arxiv.org/abs/2412.04041v1), 2024-12-05

(41) [SocialMind: LLM-based Proactive AR Social Assistive System with Human-like Perception for In-situ Live Interactions](http://arxiv.org/abs/2412.04036v1), 2024-12-05

(42) [Marco-LLM: Bridging Languages via Massive Multilingual Training for Cross-Lingual Enhancement](http://arxiv.org/abs/2412.04003v1), 2024-12-05

(43) [MTMT: Consolidating Multiple Thinking Modes to Form a Thought Tree for Strengthening LLM](http://arxiv.org/abs/2412.03987v1), 2024-12-05

(44) [Comprehensive Audio Query Handling System with Integrated Expert Models and Contextual Understanding](http://arxiv.org/abs/2412.03980v1), 2024-12-05

(45) [AI-based Attacker Models for Enhancing Multi-Stage Cyberattack Simulations in Smart Grids Using Co-Simulation Environments](http://dx.doi.org/10.1109/CSR61664.2024.10679448), 2024-12-05

(46) [Demonstration Selection for In-Context Learning via Reinforcement Learning](http://arxiv.org/abs/2412.03966v1), 2024-12-05

(47) [Chain-of-Thought in Large Language Models: Decoding, Projection, and Activation](http://arxiv.org/abs/2412.03944v1), 2024-12-05

(48) [AIpparel: A Large Multimodal Generative Model for Digital Garments](http://arxiv.org/abs/2412.03937v1), 2024-12-05

(49) [Exploring AI Text Generation, Retrieval-Augmented Generation, and Detection Technologies: a Comprehensive Overview](http://arxiv.org/abs/2412.03933v1), 2024-12-05

(50) [A Survey on Large Language Model-Based Social Agents in Game-Theoretic Scenarios](http://arxiv.org/abs/2412.03920v1), 2024-12-05

(51) [Integrating Various Software Artifacts for Better LLM-based Bug Localization and Program Repair](http://arxiv.org/abs/2412.03905v1), 2024-12-05

(52) [MISR: Measuring Instrumental Self-Reasoning in Frontier Models](http://arxiv.org/abs/2412.03904v1), 2024-12-05

(53) [Uniform Discretized Integrated Gradients: An effective attribution based method for explaining large language models](http://arxiv.org/abs/2412.03886v1), 2024-12-05

(54) [CreatiLayout: Siamese Multimodal Diffusion Transformer for Creative Layout-to-Image Generation](http://arxiv.org/abs/2412.03859v1), 2024-12-05

(55) [How Good is ChatGPT in Giving Adaptive Guidance Using Knowledge Graphs in E-Learning Environments?](http://arxiv.org/abs/2412.03856v1), 2024-12-05

(56) [Educational-Psychological Dialogue Robot Based on Multi-Agent Collaboration](http://arxiv.org/abs/2412.03847v1), 2024-12-05

(57) [A large language model-type architecture for high-dimensional molecular potential energy surfaces](http://arxiv.org/abs/2412.03831v1), 2024-12-05

(58) [Beyond the Binary: Capturing Diverse Preferences With Reward Regularization](http://arxiv.org/abs/2412.03822v1), 2024-12-05

(59) [Synergizing LLMs and Knowledge Graphs: A Novel Approach to Software Repository-Related Question Answering](http://arxiv.org/abs/2412.03815v1), 2024-12-05

(60) [EditScout: Locating Forged Regions from Diffusion-based Edited Images with Multimodal LLM](http://arxiv.org/abs/2412.03809v1), 2024-12-05

(61) [Agent AI with LangGraph: A Modular Framework for Enhancing Machine Translation Using Large Language Models](http://arxiv.org/abs/2412.03801v1), 2024-12-05

(62) [Automated Multi-Label Annotation for Mental Health Illnesses Using Large Language Models](http://arxiv.org/abs/2412.03796v1), 2024-12-05

(63) [A Contemporary Overview: Trends and Applications of Large Language Models on Mobile Devices](http://arxiv.org/abs/2412.03772v1), 2024-12-04

(64) [A Benchmark for Math Misconceptions: Bridging Gaps in Middle School Algebra with AI-Supported Instruction](http://arxiv.org/abs/2412.03765v1), 2024-12-04

(65) [Advancing Auto-Regressive Continuation for Video Frames](http://arxiv.org/abs/2412.03758v1), 2024-12-04

(66) [Enhancing IR-based Fault Localization using Large Language Models](http://arxiv.org/abs/2412.03754v1), 2024-12-04

(67) [VidHalluc: Evaluating Temporal Hallucinations in Multimodal Large Language Models for Video Understanding](http://arxiv.org/abs/2412.03735v1), 2024-12-04

(68) [A Water Efficiency Dataset for African Data Centers](http://arxiv.org/abs/2412.03716v1), 2024-12-04

(69) [Scaling Inference-Time Search with Vision Value Model for Improved Visual Comprehension](http://arxiv.org/abs/2412.03704v1), 2024-12-04

(70) [System Test Case Design from Requirements Specifications: Insights and Challenges of Using ChatGPT](http://arxiv.org/abs/2412.03693v1), 2024-12-04

(71) [Personalizing Multimodal Large Language Models for Image Captioning: An Experimental Analysis](http://arxiv.org/abs/2412.03665v1), 2024-12-04

(72) [From Individual to Society: A Survey on Social Simulation Driven by Large Language Model-based Agents](http://arxiv.org/abs/2412.03563v1), 2024-12-04

(73) [SPICE: Smart Projection Interface for Cooking Enhancement](http://arxiv.org/abs/2412.03551v1), 2024-12-04

(74) [Evaluating Gender Bias Transfer between Pre-trained and Prompt-Adapted Language Models](http://arxiv.org/abs/2412.03537v1), 2024-12-04

(75) [A Review on Scientific Knowledge Extraction using Large Language Models in Biomedical Sciences](http://arxiv.org/abs/2412.03531v1), 2024-12-04

(76) [FANAL -- Financial Activity News Alerting Language Modeling Framework](http://arxiv.org/abs/2412.03527v1), 2024-12-04

(77) [You're (Not) My Type -- Can LLMs Generate Feedback of Specific Types for Introductory Programming Tasks?](http://arxiv.org/abs/2412.03516v1), 2024-12-04

(78) [Training-Free Mitigation of Language Reasoning Degradation After Multimodal Instruction Tuning](http://arxiv.org/abs/2412.03467v1), 2024-12-04

(79) [From Words to Workflows: Automating Business Processes](http://arxiv.org/abs/2412.03446v1), 2024-12-04

(80) [RedStone: Curating General, Code, Math, and QA Data for Large Language Models](http://arxiv.org/abs/2412.03398v1), 2024-12-04

(81) [Network-aided Efficient Large Language Model Services With Denoising-inspired Prompt Compression](http://arxiv.org/abs/2412.03621v1), 2024-12-04

(82) [WiS Platform: Enhancing Evaluation of LLM-Based Multi-Agent Systems Through Game-Based Analysis](http://arxiv.org/abs/2412.03359v1), 2024-12-04

(83) [Improving Linguistic Diversity of Large Language Models with Possibility Exploration Fine-Tuning](http://arxiv.org/abs/2412.03343v1), 2024-12-04

(84) [AI-Driven Day-to-Day Route Choice](http://arxiv.org/abs/2412.03338v1), 2024-12-04

(85) [Chatting with Logs: An exploratory study on Finetuning LLMs for LogQL](http://arxiv.org/abs/2412.03612v1), 2024-12-04

(86) [Intent-driven In-context Learning for Few-shot Dialogue State Tracking](http://arxiv.org/abs/2412.03270v1), 2024-12-04

(87) [DynamicControl: Adaptive Condition Selection for Improved Text-to-Image Generation](http://arxiv.org/abs/2412.03255v1), 2024-12-04

(88) [Alignment at Pre-training! Towards Native Alignment for Arabic LLMs](http://arxiv.org/abs/2412.03253v1), 2024-12-04

(89) [Controlling the Mutation in Large Language Models for the Efficient Evolution of Algorithms](http://arxiv.org/abs/2412.03250v1), 2024-12-04

(90) [AIM: Adaptive Inference of Multi-Modal LLMs via Token Merging and Pruning](http://arxiv.org/abs/2412.03248v1), 2024-12-04

(91) [Does Safety Training of LLMs Generalize to Semantically Related Natural Prompts?](http://arxiv.org/abs/2412.03235v1), 2024-12-04

(92) [Linq-Embed-Mistral Technical Report](http://arxiv.org/abs/2412.03223v1), 2024-12-04

(93) [Survey of different Large Language Model Architectures: Trends, Benchmarks, and Challenges](http://dx.doi.org/10.1109/ACCESS.2024.3482107), 2024-12-04

(94) [ClusterKV: Manipulating LLM KV Cache in Semantic Space for Recallable Compression](http://arxiv.org/abs/2412.03213v1), 2024-12-04

(95) [U-MATH: A University-Level Benchmark for Evaluating Mathematical Skills in LLMs](http://arxiv.org/abs/2412.03205v1), 2024-12-04

(96) [Beyond Questions: Leveraging ColBERT for Keyphrase Search](http://arxiv.org/abs/2412.03193v1), 2024-12-04

(97) [Weighted-Reward Preference Optimization for Implicit Model Fusion](http://arxiv.org/abs/2412.03187v1), 2024-12-04

(98) [Automatic detection of diseases in Spanish clinical notes combining medical language models and ontologies](http://arxiv.org/abs/2412.03176v1), 2024-12-04

(99) [LLM-Mirror: A Generated-Persona Approach for Survey Pre-Testing](http://arxiv.org/abs/2412.03162v2), 2024-12-04

(100) [Byte BPE Tokenization as an Inverse string Homomorphism](http://arxiv.org/abs/2412.03160v1), 2024-12-04
